themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -2,35 +2,35 @@
   <div class="flex-container su-card__contents">
     <div class="{% if image|render|striptags('<drupal-render-placeholder> <img> <picture> <source>')|trim  %}flex-9-of-12{% endif %}">
       <div class="su-opportunity-type">
-        <div>
+        <div  aria-describedby="{{ 'Opportunity Type'|t }}">
```

Comment:
from what I read, the `aria-describedby` value should be an ID of an element that describes it. I believe this isn't the correct attribute for this value. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

Reply:
This is great to know! I've updated this template to use `aria-labelby` instead which seems more appropriate.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Reply:
Those appear to be mostly identical.
> Value: A space-separated list of element IDs

I believe you want the `aria-label` attribute https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute#:~:text=The%20aria%2Dlabel%20attribute%20is,%2C%20use%20aria%2Dlabelledby%20instead.

Reply:
>  Use it in cases where a text label is not visible on the screen.
Ah! The `aria-label` is definitely the one to use for opportunity type since it doesn't have a visible text label. Thanks Mike!